Source/Plugins/EditorModules/ObjectInspector/ObjectInspectorPlugin.cs

```diff
@@ -30,7 +24,7 @@ internal static ObjectInspectorPlugin Instance
 
 		private	List<ObjectInspector>	objViews		= new List<ObjectInspector>();
 		private	bool					isLoading		= false;
-
+		private ObjectInspectorSettings userSettings;
```

Comment:
Style: Please do an explicit value assignment to `null` for consistency and readability reasons.